maboss/simulation.py

```diff
@@ -336,3 +343,15 @@ def copy_and_update_parameters(sim, parameters):
     new_sim.update_parameters( **parameters )
     return new_sim
 
+def to_biolqm(maboss_model):
+    
+    path = tempfile.mkdtemp()
+    bnet_fd, bnet_filename = tempfile.mkstemp(dir=path, suffix='.bnet')
```

Comment:
You can use a helper function provided by colomoto-jupyter (which will automatically delete the file when the notebook stops):
```
bnet_filename = new_output_file("bnet")
```
you need
```
from colomoto_jupyter.sessionfiles import new_output_file
```
(see https://github.com/GINsim/GINsim-python/blob/master/biolqm/__init__.py for examples of model conversions)

Reply:
Done

Reply:
Done